application/src/test/resources/application-test.yml

```
@@ -0,0 +1,3 @@
+spring:
+  liquibase:
+    change-log: file:etc/database/changelog.yml
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)